src/CryptoInterface.h

```diff
@@ -193,15 +193,30 @@ int BSL_AuthCtx_Finalize(BSL_AuthCtx_t *hmac_ctx, void **hmac, size_t *hmac_len)
 int BSL_AuthCtx_Deinit(BSL_AuthCtx_t *hmac_ctx);
 
 /**
- * @todo Doxygen
+ * Deinit and free key handle data
+ * @param[in] keyhandle key handle to clear. Assumed to be allocated with `BSL_MALLOC`.
  */
-int BSL_Crypto_UnwrapKey(BSL_Data_t *unwrapped_key_output, BSL_Data_t wrapped_key_plaintext, const char *key_id,
-                         size_t aes_variant);
+int BSL_Crypto_ClearKeyHandle(void *keyhandle);
 
 /**
- * @todo Doxygen
+ * Perform key wrap
+ * @param[in] kek_handle key encryption key handle (encryption key)
+ * @param[in] aes_variant AES variant to use for encryption
+ * @param[in] cek_handle content encryption key handle (encryption data)
+ * @param[in,out] wrapped_key output wrapped key (ciphertext) bytes
+ * @param[in,out] wrapped_key_handle output wrapped key (ciphertext) handle, allocated with `BSL_MALLOC`
```

Comment:
Rather than just text, if this is marked up as `::BSL_MALLOC()` doxygen should cross-link it to the actual macro. Same applies to every other reference to a macro, enum, or function.